receipt_dynamo/receipt_dynamo/entities/__init__.py

```
@@ -4,6 +4,33 @@
 TODO: Make this import work with intellisense
 """
 
+# =============================================================================
+# Base classes and mixins for entity consolidation
+# =============================================================================
+from receipt_dynamo.entities.geometry_entity import (  # noqa: F401
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)